library/attachment-viewer/src/main/java/im/vector/lib/attachmentviewer/SwipeToDismissHandler.kt

```diff
@@ -113,11 +113,11 @@ internal fun ViewPropertyAnimator.setAnimatorListener(
         onAnimationStart: ((Animator?) -> Unit)? = null
```

Comment:
You can now remove the nullability on the `Animator` and probably update the usage

Reply:
ec037270b54baef56f5631bd81221432cb69ea24

Reply:
c17b55b874daa656a2ee295b372375adf841ad73